src/scaffoldmaker/meshtypes/meshtype_3d_lung1.py

```diff
@@ -343,67 +343,119 @@ def generateBaseMesh(cls, region, options):
                 uElementsCount1, uElementsCount2, uElementsCount3,
                 lowerRightNodeIds, upperRightNodeIds, elementIdentifier)
 
+            # Marker points
+            lungNodesetGroup = lungGroup.getNodesetGroup(nodes)
+            markerList = []
+
+            lowerLeftElementCount = (lElementsCount1 * (lElementsCount2-1) * lElementsCount3 + lElementsCount1)
+
+            leftApexGroup = findOrCreateAnnotationGroupForTerm(annotationGroups, region,
+                                                               get_lung_term("apex of left lung"))
```

Comment:
I would like all annotation groups common to all species made outside of the species-specific code. Only groups specific to one species should be made within the species-specific code, but even then I would like the annotation groups all made together at the beginning of the section.